src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -1318,6 +1319,7 @@ class AbstractProductImage(models.Model):
         help_text=_("An image with a display order of zero will be the primary"
                     " image for a product"))
     date_created = models.DateTimeField(_("Date created"), auto_now_add=True)
+    srcset = SrcSetDescriptor('original')
```

Comment:
Why would you implement it with a descriptor? The srcset is somethinig that is useful in the frontend when designing the templates. So it would make sence the desicion of what sizes the srcset should be can be determined in the template by the frontend developer, eg full width, 6 column, or 3 column etc.

If srcsets have to be defined on classes like this using the descriptor, we would NEED to override the backend python classes when we need something changed in the frontend. I think that is too complex.

Am I seeing this correctly?